CMakeLists.txt

```diff
@@ -158,7 +158,6 @@ if(BUILD_TESTING)
     EXECUTABLE test_component)
 
   install(TARGETS ${PROJECT_NAME}_test_component_lib
-    EXPORT export_${PROJECT_NAME}
```

Comment:
I don't know why this was originally exported, but it could also be exported as `export_${PROJECT_NAME}_test` or something to prevent accidental linking against? Although to be fair, there isn't a reason to install (or export) the test library.

Reply:
yeah, installing or exporting the library sounds like a bad idea.
The workaround that rclcpp_components is using looks better, though this seems fine to solve the CI issues.